cmd/params/params.go

```diff
@@ -749,7 +749,7 @@ func (p API) String() string {
 	apiKey := p.Key
 	if len(apiKey) > 4 {
 		// only show last 4 chars of api key in logs
-		apiKey = "..." + apiKey[:len(apiKey)-4]
```

Comment:
Why change `apiKey[:len(apiKey)-4]` to `apiKey[len(apiKey)-4:]`? They both return the last 4 characters of the string. Is one better than the other?

Reply:
https://go.dev/play/p/gBW5RAK07QO

Reply:
Oops! Thanks for catching that!